week_1/week1.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <link rel="stylesheet" type="text/css" href="week1.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Week_1</title>
+</head>
+<body>
+    <h1>MY GUEST BOOK</h1>
+    <h3>Please sign my guest book. Thanks! </h3>
+    <div class="user">
+
+        <div class="text">
+         <label for="name"class= myname>What is your Name? </label> <input type="text">

```

Comment:
`for` specifies the id of the form element the label should be bound to
```suggestion
         <label for="user-name"class= myname>What is your Name? </label> <input id="user-name" type="text">

```